create-vite-express/src/patches.ts

```diff
@@ -1,16 +1,17 @@
-export const PATCHES: Record<string, (content: string) => string> = {
-  "package.json": (content: string) => {
-    console.log(process.platform);
+export const PATCHES: Record<string, (content: string, variables: Array<String>) => string> = {
```

Comment:
Maybe we could achieve more type-safety by passing variables as a typed object instead of just an array of strings?

Reply:
> Maybe we could achieve more type-safety by passing variables as a typed object instead of just an array of strings?

I changed it